app/views/news_updates/_form.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= edit_form_for(news_update) do |f| %>
+    <%= f.input :message, hint: "Use HTML for formatting" %>
+    <%= f.label :duration_in_days, "Duration:" %>
+    <p><%= f.number_field :duration_in_days, label: "Days", min: 1, max:30, value: news_update.duration.in_days.to_i %> days</p>
```

Comment:
The problem with a collection imo is that it's really annoying from an user's perspective to have to select the number instead of typing it. It's faster to type 12 than to type 12 AND have to select it from the dropdown.